src/Native/System.Native/pal_io.h

```diff
@@ -610,3 +621,11 @@ extern "C" void Sync();
  * Returns the number of bytes written on success; otherwise, returns -1 and sets errno
  */
 extern "C" int32_t Write(int32_t fd, const void* buffer, int32_t bufferSize);
+
+
+/**
+* Gets the windows size of the terminal
+*
+* Returns 0 on success; otherwise, returns errorNo.
+*/
+extern "C" int32_t GetWindowSize(WinSize* windowsSize);
```

Comment:
Really?.. I did not get compilation error. But, it could be because I copy-pasted my changes from dev box to a unix box and it might have taken care of this automatically. Changing it now.